tests/coalaCITest.py

```diff
@@ -137,3 +137,17 @@ def test_fail_acquire_settings_debug(self):
                 r'were not provided.'
         ):
             self.test_fail_acquire_settings(debug=True)
+
+    def test_limit_files(self):
+        with bear_test_module(), \
+             prepare_file(['\t#include <a>'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '--non-interactive',
+                '-c', os.devnull,
+                '--limit-files', 'some_file_name',
```

Comment:
shouldn't this be `filename` so that it actually tests whether or not `--limit-files` is considered?

also, there should be an assert checking if the `\t` was reported; it should be converted to spaces because of `use_spaces=True`, so there will be some logging to console

Reply:
No, it shouldn't be filename AFAIK. The test shows, that if `-f` args do not match `--limit-files` pattern, files won't be put into coala

Reply:
This test is only about `--limit-files`, `\t` report is being checked in different tests

Reply:
Ah, my mistake.

One tiny review: it's `--limit-files`, not `--limit--files` in your commit shortlog :)

Reply:
You are not testing that `--limit-files` does what people use it for.
The only conclusion that is certain from this test, alone, is that using `--limit-files` results in zero results.